### PR TITLE
override get_config for transformer block and change dropout layer name

### PR DESCRIPTION
**Pull request by Proinn**
Hi,

This change overrides the get_config method of the TransformerBlock custom layer so the ViT models can be loaded with tensorflow.keras.models.load_model. Also changed the name of the dropout layer in this custom layer so it doesn't override the dropout rate anymore

I hope this is helpful!

Proinn

### CODE REVIEW

**Review by faustomorales — APPROVED** [review]
Nice work, thank you!

**Comment by faustomorales** [comment]
This is now included in v0.0.15.
